### PR TITLE
Add keepalive configs as templated strings and modify their values 

### PR DESCRIPTION
**Pull request by JDPrabasha**
### Purpose
Lower the 5-minute interval to check for liveness of adapter after establishing a connection to it to 1 minute. This would allow for faster detection when the local adapter goes down and enable faster switching to another replica. Additionally, by making these values dynamically configurable, we can adjust them as and when needed in the future.

### Issues

Fixes https://github.com/wso2-enterprise/choreo/issues/32445

### Automation tests
 - Unit tests added: No
 - Integration tests added: No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [ ] Assigned 'Type' label
- [ ] Assigned the project
- [ ] Validated respective github issues
- [ ] Assigned milestone to the github issue(s)

### CODE REVIEW

File: router/src/main/resources/envoy.yaml.template [hunk 1]
```diff
@@ -42,9 +42,9 @@ static_resources:
       connect_timeout: 1s
       upstream_connection_options:
         tcp_keepalive:
-          keepalive_probes: 3
-          keepalive_time: 300
-          keepalive_interval: 30
+          keepalive_probes: '${KEEPALIVE_PROBES}'
```
Comment (renuka-fernando) [inline router/src/main/resources/envoy.yaml.template:45]: Shall we prefix ADAPTER to make this config more specific?
```suggestion
          keepalive_probes: '${ADAPTER_KEEPALIVE_PROBES}'
```

Comment (renuka-fernando) [reply router/src/main/resources/envoy.yaml.template:45]: I guess we can keep this as it is and use it in all places. Hence +1

**Review by renuka-fernando — DISMISSED** [review]

File: router/src/main/resources/Dockerfile [hunk 2]
```diff
@@ -47,6 +47,10 @@ ENV ROUTER_LABEL="Default"
 ENV ROUTER_PRIVATE_KEY_PATH=/home/wso2/security/keystore/mg.key
 ENV ROUTER_PUBLIC_CERT_PATH=/home/wso2/security/keystore/mg.pem
 
+ENV KEEPALIVE_PROBES=3
+ENV KEEPALIVE_TIMEOUT=60
```
Comment (mevan-karu) [inline router/src/main/resources/Dockerfile:51]: Shall we keep the current default as it is and identify a proper value in a lower environment before sending this to higher environments.

Comment (JDPrabasha) [reply router/src/main/resources/Dockerfile]: Reverted values in ac33a9c33082062051a8f6c5f2685e743f408b0b

**Review by renuka-fernando — APPROVED** [review]

**Comment by choreo-cicd** [comment]
[failed] Dataplane(NorthEU) cluster : dev-deployment-v2 : 20241217.12

**Comment by choreo-cicd** [comment]
[succeeded]  : dev-deployment-v2 : 20241217.12

**Comment by choreo-cicd** [comment]
[failed] Dataplane(EastUS) cluster : dev-deployment-v2 : 20241217.12

**Comment by choreo-cicd** [comment]
[failed] Dataplane(EastUS) cluster : dev-deployment-v2 : 20241217.13

**Comment by choreo-cicd** [comment]
[failed] Dataplane(NorthEU) cluster : dev-deployment-v2 : 20241217.13

**Comment by choreo-cicd** [comment]
[failed] Dataplane(EastUS) cluster : dev-deployment-v2 : 20241217.14

**Comment by choreo-cicd** [comment]
[failed] Dataplane(NorthEU) cluster : dev-deployment-v2 : 20241217.14

**Comment by choreo-cicd** [comment]
[failed] Dataplane(NorthEU) cluster : dev-deployment-v2 : 20241217.15

**Comment by choreo-cicd** [comment]
[failed] Dataplane(EastUS) cluster : dev-deployment-v2 : 20241217.15

**Comment by choreo-cicd** [comment]
[failed] Dataplane(EastUS) cluster : dev-deployment-v2 : 20241217.16

**Comment by choreo-cicd** [comment]
[failed] Dataplane(NorthEU) cluster : dev-deployment-v2 : 20241217.17

**Comment by choreo-cicd** [comment]
[failed] Dataplane(EastUS) cluster : dev-deployment-v2 : 20241217.17

**Comment by choreo-cicd** [comment]
[failed] Dataplane(EastUS) cluster : dev-deployment-v2 : 20241217.18

**Comment by choreo-cicd** [comment]
[failed] Dataplane(NorthEU) cluster : dev-deployment-v2 : 20241217.18

**Comment by choreo-cicd** [comment]
[failed] Dataplane(NorthEU) cluster : dev-deployment-v2 : 20241217.19

**Comment by choreo-cicd** [comment]
[failed] Dataplane(NorthEU) cluster : dev-deployment-v2 : 20241217.21

**Comment by choreo-cicd** [comment]
[failed] Dataplane(NorthEU) cluster : dev-deployment-v2 : 20241217.22

**Comment by choreo-cicd** [comment]
[]  : dev-deployment-v2 : 20241218.2

**Comment by choreo-cicd** [comment]
[] Dataplane(NorthEU) cluster : dev-deployment-v2 : 20241218.2

**Comment by choreo-cicd** [comment]
[] Dataplane(EastUS) cluster : dev-deployment-v2 : 20241218.2

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(EastUS) cluster : prod-deployment-v2 : 20241219.1

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : prod-deployment-v2 : 20241219.1

**Comment by choreo-cicd** [comment]
[failed] Dataplane(CDP2-WU2) cluster : prod-deployment-v2 : 20241219.1

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : prod-deployment-v2 : 20241219.1

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(NorthEU) cluster : prod-deployment-v2 : 20241219.1

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(EastUS) cluster : prod-deployment-v2 : 20241219.1

**Comment by choreo-cicd** [comment]
[] Dataplane(EastUS) cluster : prod-deployment-v2 : 20241219.2

**Comment by choreo-cicd** [comment]
[] Dataplane(NorthEU) cluster : prod-deployment-v2 : 20241219.2

**Comment by choreo-cicd** [comment]
[] Controlplane cluster : prod-deployment-v2 : 20241219.2

**Pull request merged**